examples/next/docs/vue3/demo/src/hooks-callbacks.ts

```diff
@@ -0,0 +1,103 @@
+import Handsontable from 'handsontable';
+
+import {
+  SELECTED_CLASS,
+  ODD_ROW_CLASS
+} from './constants';
+
+type AddClassesToRows = (
+  TD: HTMLTableCellElement,
+  row: number,
+  column: number,
+  prop: number | string,
+  value: any,
+  cellProperties: Handsontable.CellProperties
+) => void;
+
+export const addClassesToRows: AddClassesToRows = (
+  TD,
+  row,
+  column,
+  prop,
+  value,
+  cellProperties
+) => {
+  // Adding classes to `TR` just while rendering first visible `TD` element
+  if (column !== 0) {
+    return;
+  }
+
+  const parentElement = TD.parentElement;
+
+  if (parentElement === null) {
+    return;
+  }
+
+  const rowData = cellProperties.instance.getSourceDataAtRow(row) as { checked: boolean };
```

Comment:
You can use the `RowObject` type in here (and a couple of lines below).
It's available for importing from:
```
import type { RowObject } from 'handsontable/types/common';
```
```suggestion
  const rowData: RowObject = cellProperties.instance.getSourceDataAtRow(row);
```